app/templates/user_documentation.html

```diff
@@ -7,29 +7,96 @@ <h1 class="header">Help Menu</h1>
-	<p class = "helpP"><span class="titleHead" id="helpP">How do I view the chart generated by the clustering algorithm?</span></p>
-	<p class = "helpP"><span class="titleResponse">You can view the chart generated by the clustering algorithm by clicking on the green &ldquo;Process&rdquo; button and then scrolling down to the Chart section.</span></p>
-	<p class = "helpP"><span style="overflow: hidden; display: inline-block; margin: 0.00px 0.00px; border: 0.00px solid #000000; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px); width: 624.00px; height: 440.00px;"><img alt="" src={% static "images/image4.png"%} style="width: 624.00px; height: 440.00px; margin-left: 0.00px; margin-top: 0.00px; transform: rotate(0.00rad) translateZ(0px); -webkit-transform: rotate(0.00rad) translateZ(0px);" title=""></span></p>
+<body>
+		<div class="help-div">
+			<button type="button" class="help-title titleHead" onclick="handleHelpTitleClick('response1')">What is this website and why was this website created?</button>
+			<div class="help-response titleResponse" id="response1">
+				<p>
+					Astronomy researchers have difficulty visualizing the large heterogeneous datasets of binary stellar trajectories. 
+					With the vast number of publicly available datasets retrieved from astronomical observations and simulation, 
+					there needs to be a tool that allows researchers to quickly search for patterns in the data. 
+					The main use case for this data is the clustering of binary stars based on the proximity and diversity of the 
+					stellar attributes across multiple distance functions. Using these distance functions, analysts would like to measure the outcome 
+					of different attributes related to the attribute&rsquo;s proximity vs. diversity. To solve this problem, we have developed an 
+					intuitive web-based tool that allows users to select distance functions for different data parameters. 
+					This application enables users to query a large heterogeneous dataset with a focus on proximity vs. diversity. 
+					In all, this website allows users to select a database, select 3 specific attributes within that dataset in relation to proximity and diversity, 
+					attach a weight to each attribute, select a time that the algorithm will cluster at, and select the number of clusters they are looking for. 
+					Once this is complete, a graph will be generated that represents the clustering of the attributes selected.
+				</p>
+			</div>
+		</div>
+		<div class="help-div">
+			<button type="button" class="help-title titleHead" onclick="handleHelpTitleClick('response2')">What is proximity and diversity?</button>
```

Comment:
For all buttons `onClick`s:
Pass `this`, and inside the function, use `querySelector` to get the sibling (help-response div). Or just use `nextElementSibling`.